pkg/Library/CoreXenon/Designer/Properties.js

```diff
@@ -53,20 +52,27 @@ const updateDataProp = (design, propId, value) => {
 const updatePropWithConnection = async (design, objectId, propId, value) => {
   const graph = design.graph;
   const connections = graph.connections ??= nob();
-  if (value.connection?.value?.length > 0) {
-    const connValue = value.connection.value;
+  const regenerateBindings = () => {
```

Comment:
nit - maybe this could be a `Binder` method instead?

PS: this is just me demonstrating how carefully I review :)

Reply:
Makes sense.